src/app/applications/applications.controller.listing.js

```diff
@@ -0,0 +1,100 @@
+(function() {
+    'use strict';
+
+    // Declare the controllers first
+    angular
+        .module( 'roboconf.applications' )
+        .controller( 'ApplicationsListingController', applicationsListingController );
+
+    // Declare the controller functions then
+    // Specify the injection to prevent errors on minification
+    applicationsListingController.$inject = [ 'Restangular', '$scope', 'rUtils', 'rShare', '$route', 'rAppTemplates' ];
+    
+    // Then comes the function
+    function applicationsListingController( Restangular, $scope, rUtils, rShare, $route, rAppTemplates ) {
+    	
+    	// Fields
+    	$scope.isTpl = $route.current.tpl;
+    	$scope.invoked = false;
+    	$scope.error = false;
+    	$scope.apps = [];
+    	$scope.searchFilter = '';
+    	$scope.selectedApp = null;
+    	
+    	// Functions declaration
+    	$scope.showApplication = showApplication;
+    	$scope.hideApplication = hideApplication;
+    	$scope.findBlockClass = findBlockClass;
+    	
+    	// Initial actions
+    	initializeSearch();
+    	
+    	if( $scope.isTpl ) {
+    		listApplicationTemplates();
+    	} else {
+    		listApplications();
+    	}
+    	
+    	var tmp = rShare.eatLastItem();
+    	if( tmp ) {
+    		$scope.selectedApp = tmp;
+    		rUtils.showRightBlock( 200 );
+    	}
+    	
+    	// Function definitions
+    	function showApplication( app, t ) {
+    		$scope.selectedApp = app;
+    		rUtils.showRightBlock( t );
+        }
+    	
+    	function hideApplication() {
+    		$scope.selectedApp = null;
+    		rUtils.hideRightBlock();
+        }
+    	
+    	function findBlockClass( app ) {
```

Comment:
Angular has _ngClass_ for this: https://docs.angularjs.org/api/ng/directive/ngClass. Please use this.